eldritch/mythos.py

```diff
@@ -471,13 +643,195 @@ def get_modifier(self, thing, attribute):
+
+  def get_override(self, thing, attribute):
+    if isinstance(thing, items.Spell) and attribute == "can_use":
+      return False
+    return None
+
+
+class Mythos52(Environment):
+  def __init__(self):
+    super().__init__("Mythos52", "Cave", "Roadhouse", {"plus"}, {"crescent"}, env_type="weather")
+
+  def create_event(self, state) -> events.Sequence:
+    seq = super().create_event(state)
+    seq.events.extend([
+        events.ReturnMonsterFromBoard(None, monster)
+        for monster in state.monsters
+        if isinstance(monster.place, places.CityPlace) and monster.name == "Haunter"
+    ])
+    return seq
+
+  def get_modifier(self, thing, attribute, state):
+    if isinstance(getattr(thing, "place", None), places.CityPlace):
+      if attribute == "sneak":
+        return -1
+      if attribute == "will":
+        return 1
+    return 0
+
+  def get_trigger(self, event, state):
+    # TODO: Override Haunter deployment
+    return None
+
+
+class Mythos53(Headline):
+  def __init__(self):
+    super().__init__("Mythos53", "Square", "Unnamable", {"hex"}, {"slash", "triangle", "star"})
+
+  def create_event(self, state) -> events.Sequence:
+    seq = super().create_event(state)
+    seq.events.extend(ReturnAndIncrease({"Land Squid", "Giant Worm"}))
+    return seq
+
+
+# TODO: Mythos 54, may return from Other Worlds
+
+class Mythos55(Environment):
+  def __init__(self):
+    super().__init__("Mythos55", "Isle", "Science", {"plus"}, {"moon"}, "mystic")
+
+  def get_modifier(self, thing, attribute, state):
+    if (isinstance(thing, monsters.Monster)
+       and thing.has_attribute("undead", state, None)
+       and attribute == "toughness"):
+      return 1
+    return 0
+
+
+class Mythos56(Headline):
+  def __init__(self):
+    super().__init__("Mythos56", "WitchHouse", "Cave", {"moon"}, {"plus"})
+
+  def create_event(self, state) -> events.Sequence:
+    seq = super().create_event(state)
+    seq.events.append(events.ReleaseMonstersToLocation("Northside", 2))
+    return seq
+
+
+class Mythos57(Headline):
+  def __init__(self):
+    super().__init__("Mythos57", "WitchHouse", "Cave", {"hex"}, {"slash", "triangle", "star"})
+
+  def create_event(self, state) -> events.Sequence:
+    seq = super().create_event(state)
+    seq.events.extend([
+        events.CloseLocation("Roadhouse", for_turns=1),
+        events.AddGlobalEffect(self, source_deck="mythos", active_until=state.turn+1),
+    ])
+    for char in state.characters:
+      whiskies = [pos for pos in char.possessions if pos.name == "Whiskey"]
+      if not whiskies:
+        continue
+      check = events.Check(char, "sneak", -1)
+      pass_fail = events.PassFail(
+          char, check,
+          events.Nothing(),
+          events.Sequence(
+              [events.Arrested(char)]
+              + [events.DiscardSpecific(char, whiskey) for whiskey in whiskies],
+              char)
+      )
+      char_seq = events.Sequence([check, pass_fail], char)
```

Comment:
When the second argument to PassFail is an event, PassFail automatically returns a sequence. You do not need to create your own.